front/src/Phaser/Game/GameScene.ts

```diff
@@ -957,9 +957,11 @@ export class GameScene extends DirtyScene {
 
         this.gameMap.onPropertyChange(GameMapProperties.JITSI_ROOM, (newValue, oldValue, allProps) => {
             if (newValue === undefined) {
+                this.activatablesManager.enableSelectingByDistance();
                 layoutManagerActionStore.removeAction("jitsi");
                 this.stopJitsi();
             } else {
+                this.activatablesManager.disableSelectingByDistance();
                 const openJitsiRoomFunction = () => {
```

Comment:
I'm not sure this is the best place to do this.

The issue is not really with Jitsi, but with any "action" that can be triggered using the SPACE key.
For instance, such actions can be triggered by "openWebsiteTrigger" or even by the API.

If you look line 986, we are using a "layoutManagerActionStore" that is in charge of displaying the "action" text.

I'm pretty sure that the calls to `this.activatablesManager.enableSelectingByDistance();` and `this.activatablesManager.disableSelectingByDistance();` should actually be put somewhere in this "layoutManagerActionStore".